### DISCUSSION TITLE
posts/yaoyuandejiushizhu/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#54] [Comments]
# posts/yaoyuandejiushizhu/

豆豆. 遥远的救世主[M]. 作家出版社, 2005.
[&hellip;] 肖亚文说：“认识这个人就是开了一扇窗户，就能看到不一样的东西，听到不一样的声音，能让你思考、觉悟，这已经够了。其它还有很多，比如机会、帮助，我不确定。这个在一般人看来可能不重要，但我知道这个人很重要。”
[&hellip;] 肖亚文说：“当你觉得这个人很特别的时候，千万别对这种人动心思，一旦动了那种心思你就算把地狱之门打 &hellip;

https://shitao5.org/posts/yaoyuandejiushizhu/

**Comment by LFG12** [comment]
大赞大赞呀涛哥，我是看过这个电视剧《天道》，还没来及看书，看完后心里亦是久久不能平静
想请问下涛哥做原文摘录有什么好方法吗，我习惯于看纸质书，有时想去博客里分享读书记录去手敲就比较麻烦

**Reply by Shitao5** [reply]
刘同学好，《遥远的救世主》很精彩，我通宵了两天看完了。

我目前几乎不看纸质书了，以前看纸质书的时候是对着书手敲的。我打字速度还算可以，另外把手敲摘录当做加深记忆的一种途径，所以没觉得麻烦。目前我更多看的是电子书，刚刚你的问题启发我去把一篇很古早、未完成的博客翻出来加工下发表了，如有兴趣可以瞅一眼：[如何同时阅读多本书](https://shitao5.org/posts/multiple-books/)。
